src/runrms/executor/_rms_executor.py

```diff
@@ -22,85 +21,45 @@ class RMSExecutor(ABC):
 
     def __init__(self, config: InteractiveRMSConfig | FMRMSConfig) -> None:
         self._config = config
+        self._exec_env = self._init_exec_env()
 
-        # We want to append to or carry forward the following env vars
-        pre_env = {}
-        pre_env["PYTHONPATH"] = os.environ.get("PYTHONPATH", None)
-        pre_env["RMS_PLUGINS_LIBRARY"] = os.environ.get("RMS_PLUGINS_LIBRARY", None)
```

Comment:
This used to be set with `SETENV` in ERT user configurations because the prior RMS forward model, when included in ERT, did not store this in the separate site configuration. It was preserved for that reason, but with the site deployment set-up we are moving into now we should not honor it any longer as it will point to out-of-date plugins.